src/languages.js

```diff
@@ -57,17 +57,9 @@ export const DEFAULT_LANGUAGE_LOCALE = "en";
  */
 export const getDefaultLanguageLocale = (
   fallbackLocale: string = DEFAULT_LANGUAGE_LOCALE,
-) => {
-  const { localeIdentifier, preferredLanguages } = Locale.constants();
-  const locale =
-    (preferredLanguages && preferredLanguages[0]) || localeIdentifier;
-  const matches = locale.match(/([a-z]{2,4}[-_]([A-Z]{2,4}|[0-9]{3}))/);
-  const lang = (matches && matches[1].replace("_", "-")) || "en-US";
-  return (
-    fullySupportedLocales.find(locale => lang.startsWith(locale)) ||
-    fallbackLocale
-  );
-};
+) =>
+  RNLocalize.findBestAvailableLanguage(fullySupportedLocales)?.languageTag ||
+  fallbackLocale;
```

Comment:
- Adapting to the change of library:
 - **`src/languages.js`**:
     The default app language can now be obtained directly from `react-native-localize` with `findBestAvailableLanguage(listOfSupportedLanguages)`.